pipelines/internal/commands/run/run.go

```diff
@@ -185,11 +185,21 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 		return nil
 	}
 
+	return runPipeline(ctx, service, req)
+}
+
+func runPipeline(ctx context.Context, service *genomics.Service, req *genomics.RunPipelineRequest) error {
+	if *attempts == 0 && *pvmAttempts == 0 {
```

Comment:
Let's pull this out and put it with the earlier dryRun check:

if *dryRun || (*attempts == 0 && *pvmAttempts == 0) {
   return nil
}